web/impact/impact/v1/helpers/user_helper.py

```diff
@@ -61,9 +62,10 @@
     "company": EXPERT_STRING_FIELD,
     "title": EXPERT_STRING_FIELD,
     "bio": NON_MEMBER_STRING_FIELD,
+    "primary_industry": EXPERT_INDUSTRY_FIELD,
     "primary_industry_id": PRIMARY_INDUSTRY_ID_FIELD,
```

Comment:
does this field (primary_industry_id) need to exist anymore?

Reply:
Yes, for PATCH and POST.